File: A1softechTaskManagementEmployees/ViewModels/Employee.cs
using System.ComponentModel.DataAnnotations;

namespace A1softechTaskManagementEmployees.ViewModels
{
    public class VMEmployee
    {
        public Guid ID { get; set; } = new Guid();
        public string Name { get; set; }
        public string Mobile { get; set; }
        [EmailAddress]
        public string Email { get; set; }
        public float Salary { get; set; }
        public List<VMEmployeeTaxs> EmployeeTaxs { get; set; } = new();
    }
}

File: A1softechTaskManagementEmployees/client/src/app/models/employee.ts
import { Guid } from "guid-typescript";

export class EmployeeTaxs{
  ID: Guid ;
  Tax: number;
  EmployeeID: Guid;
  NetSalary: number;
}
export class VMEmployeeTaxs{
  id: string ;
  tax: number;
  employeeID: string;
  netSalary: number;
}

export class Employee {
  ID: Guid;
  Name: string;
  Mobile: string;
  Email: string;
  Salary: Number;
  // NetSalary: number;
  EmployeeTaxs: Array<EmployeeTaxs> = new Array<EmployeeTaxs>();
}
export class ModelResult{
  result:string;
  errorCount :number;
  message :string;
}
export function getGUID():Guid
{
  return Guid.create()
}